client/README.md

```diff
@@ -66,17 +66,26 @@ const (
 
 func main() {
 	// Make client
-	c := client.NewHTTPClient(client.HTTPConfig{
+	c, err := client.NewHTTPClient(client.HTTPConfig{
 		Addr: "http://localhost:8086",
 		Username: username,
 		Password: password,
 	})
+	
+	if err != nil {
+        	log.Println("Error: ", err)
+    	}
+
 
 	// Create a new point batch
-	bp := client.NewBatchPoints(client.BatchPointsConfig{
+	bp, err := client.NewBatchPoints(client.BatchPointsConfig{
 		Database:  MyDB,
 		Precision: "s",
 	})
+	
+	if err != nil {
+        	log.Println("Error: ", err)
```

Comment:
If you're going to include error handling in the example it's probably best to change `log.Println` to `log.Fatal` or similar.